build_tools/packaging/linux/packaging_utils.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import json
```

Comment:
I would leave the shebang off of the file if the file does not have a `if __name__ == "__main__":`
```suggestion
import json
```